src/Tests/Toolchain/ToolchainProvider.test.ts

```diff
@@ -99,16 +66,30 @@ suite('Toolchain', function() {
 
   suite('ToolchainNode', function() {
     suite('#constructor()', function() {
-      test('is constructed with params using dummy_backend', function() {
+      test('is constructed with params using toolchian_node', function() {
         const label = 'backend_node';
-        const backend = 'dummy_backend';
         const toolchain =
             new DebianToolchain(new ToolchainInfo('npm', 'package manager for Node.js'));
         const collapsibleState = vscode.TreeItemCollapsibleState.None;
-        let node = new ToolchainNode(label, backend, toolchain);
+        let node = new ToolchainNode(label, backendName, toolchain);
+        assert.strictEqual(node.label, label);
+        assert.strictEqual(node.collapsibleState, collapsibleState);
+        assert.strictEqual(node.backendName, backendName);
+      });
+    });
+
+    suite('#constructor()', function() {
```

Comment:
```suggestion
```

Suite `#constructor()` is defined multiple times. How about to merge them for consistency? (And there are other duplicated suites. e.g. `createToolchainNodes()`)